Week 05/README.md

```diff
@@ -206,17 +206,24 @@ if (!std::cin)
 	return 4;
 }
 
+// sum of array elements
 int sum = 0;
-size_t indexFound = -1; // searches for last index with the number
```

Comment:
As I might have been unclear in the previous comment, I would appreciate to keep the original logic of simultaneous summation and searching (thus keeping the indexFound variable, but not setting the initial value to -1 due to type mismatch - this is still valid approach but will result in implicit casting and consecutive compiler warning on comparison) - as with the current changes, that could be accomplished in separate loop just to make an example of that